Better siemens calculations

Kell-E opened the pull request:
Fixes accessories not being calculated in siemens calculations.

Prior to this, wearing a low-profile tactical plate wouldn't help at all if shot by stuns. This remedies the issue.

:cl: Yvesza
tweak: Accessories are now included in Siemens calculations
/:cl:

MarinaGryphon commented on code/modules/mob/living/carbon/human/human_defense.dm:
```diff
@@ -90,8 +89,13 @@ meteor_act
+	. = list()
 	var/list/clothing_items = list(head, wear_mask, wear_suit, w_uniform, gloves, shoes) // What all are we checking?
 	for(var/obj/item/clothing/C in clothing_items)
+		if (C.accessories.len)
```
This is redundant and can safely be removed. A for loop over an empty list or even `null` is perfectly fine and will simply just not run.

MarinaGryphon commented on code/modules/mob/living/carbon/human/human_defense.dm:
```diff
@@ -90,8 +89,13 @@ meteor_act
 
 	var/siemens_coefficient = max(species.siemens_coefficient,0)
 
+	. = list()
```
This isn't used anywhere.

Kell-E replied:
Thanks! Missed it when I was PRing this.

MistakeNot4892 commented on code/modules/mob/living/carbon/human/human_defense.dm:
```diff
@@ -90,8 +89,13 @@ meteor_act
 
 	var/siemens_coefficient = max(species.siemens_coefficient,0)
 
+	. = list()
```
Why are you setting this?

Kell-E replied:
Intended to remove it before the PR, slipped my mind.

Kell-E committed:
Better siemens calculations

Electrical damage now calculates all layers of clothing on an organ, including accessories.
Better Siemens calculations

Fixes accessories not being calculated in siemens calculations.
Fixes based upon review
